app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -115,6 +128,25 @@ def process_one_login_identity_verification_callback(core_identity_jwt)
     )
   end
 
+  def process_one_login_return_codes_callback
+    journey_session.answers.assign_attributes(
+      identity_confirmed_with_onelogin: false,
+      onelogin_idv_at: Time.now
+    )
+    journey_session.save!
+
+    one_login_return_codes.each do |code|
+      Stat.create!(one_login_return_code: code)
```

Comment:
```suggestion
      Stat.create!(one_login_return_code: code, journey_session: journey_session)
```
Feel like we probably want to associate these with the session they came from, always end up needing this kind of info for the inevitable "what % of users with return code X went on to successfully complete a claim"

Reply:
If it’s easy to associate then we can do it as may be useful, but in this case I expected the journey session would be purged as it will not relate to a submitted claim until we allow the user to continue.